cmd/root.go

```diff
@@ -104,8 +115,40 @@ func NewRootCmd(ctx context.Context) *cobra.Command {
 		newCmdUninstall(),
 		newCmdDelete(config),
 		newCmdWatch(config),
-		version.NewVersionCommand(),
+		cliversion.NewVersionCommand(),
 	)
 
 	return cmd
 }
+
+func versionCheck(cmd *cobra.Command) {
+	if isatty.IsTerminal(os.Stdout.Fd()) {
+		if len(cliversion.Version) != 0 && cliversion.Version != "dev" {
+			ghclient, err := utils.NewGithubClient("", nil)
```

Comment:
Rate limiting will presumably just drop out silently along with any other http errors?

Reply:
If it kicks in it will be ignored